nototools/noto_cmap_reqs.py

```diff
@@ -435,7 +451,7 @@ def _reassign_common_by_block(cmap_ops):
     'Enclosed Alphanumerics': 'Zsym',
     'Box Drawing': 'MONO',
     'Block Elements': 'MONO',
-    'Geometric Shapes': 'Zsym',
+    'Geometric Shapes': 'SYM2', # change
```

Comment:
Yes, these all are internal codes that just happen to look a lot like script codes.  I've thought about trying to rename them so that it is clear they are not really script codes, but there's stuff all over the place that assumes that the string is a script code except for a few exceptions, for example to get the English name of the 'script' from the cldr data...  To make things worse, these are also part of a font family code, which looks a lot like a bcp47 locale string, except it's not and has some extra stuff in it, again in only a few exceptional cases.  Anyway, yes, you're right to wonder about this, and I've made attempts in the past to change it, but haven't managed to because in the end the existing mostly-script-codes-plus-a-few-fake-codes seem to work well enough and I have to change all kinds of code to clean this up.  I think we just remain careful.  I use all caps for the fake codes and don't always restrict to 4 letters so collisions should be unlikely  (though BCP47 isn't case-sensitive so it's still possible).